### DISCUSSION TITLE
custom some port's triplets  in `vcpkg.json`

### DISCUSSION THREAD

**Discussion by nocanstillbb** [microsoft/vcpkg#35263] [General]
some time ,    there are  some port only can build as  `static library` or `dynamic library`,   i want to set diffrent triplets to them , how to do that?

**Comment by Osyotr** [comment]
It's not clear what you mean but I guess you are looking for per-port customization https://learn.microsoft.com/en-us/vcpkg/users/triplets#per-port-customization

**Reply by nocanstillbb** [reply]
you are right, it's 

sorry and thanks
